pop/pop_lib.c

```diff
@@ -51,45 +51,70 @@
 #endif
 
 /* These Config Variables are only used in pop/pop_lib.c */
+char *C_PopOauthRefreshCommand; ///< Config: (pop) External command to generate OAUTH refresh token
+char *C_PopPass; ///< Config: (pop) Password of the POP server
 unsigned char C_PopReconnect; ///< Config: (pop) Reconnect to the server is the connection is lost
+char *C_PopUser; ///< Config: (pop) Username of the POP server
+
+/**
+ * pop_get_field - Get connection login credentials - Implements ::ca_get_field_t
+ */
+const char *pop_get_field(enum ConnAccountField field)
+{
+  switch (field)
+  {
+    case MUTT_CA_USER:
+      return C_PopUser;
+    case MUTT_CA_PASS:
+      return C_PopPass;
+    case MUTT_CA_OAUTH_CMD:
+      return C_PopOauthRefreshCommand;
+    case MUTT_CA_HOST:
+    case MUTT_CA_LOGIN:
```

Comment:
Should this return C_PopUser, too?

Reply:
Hmm... good spot.  This is a bit more interesting.
My first draft of the code had `MUTT_CA_LOGIN` return `C_PopUser`.
The first time I tested the Pop changes, it hung for ~5 seconds, then succeeded.

The 'login' field is only used by SASL.
For me, sending 'flatcap' causes a hang, sending `NULL` works instantly.

I'll investigate further...

Reply:
Argh!  `ETOOMUCHCONFIG`
Login now returns `C_PopUser` as it should.